### PR TITLE
Server/basic

### PR DESCRIPTION
**Pull request by dleangen**
My attempts at moving closer to having a working "Basic Server"

### CODE REVIEW

File: conf/mailetcontainer.xml [hunk 1]
```diff
@@ -1,149 +0,0 @@
...
-    <processor state="rrt-error" enableJmx="false">
-      <mailet match="All" class="ToRepository">
-        <repositoryPath>file://var/mail/rrt-error/</repositoryPath>
-        <passThrough>true</passThrough>
-      </mailet>
-      <mailet match="IsSenderInRRTLoop" class="Null"/>
-      <mailet match="All" class="Bounce"/>
-    </processor>
-
-    <processor state="do-nothing-mailet" enableJmx="false">
```
Comment (ieugen) [inline conf/mailetcontainer.xml:138]: You removed my custom mailets.

File: docker-compose.yaml [hunk 2]
```diff
@@ -19,10 +19,6 @@ services:
...
       - JAVA_OPTS=-Dlogback.configurationFile="/opt/apache-james/james-server/conf/logback.xml" -Dworking.directory="/opt/apache-james/james-server"
     ports:
-      - 25
-      - 110
-      - 143
-      - 465
-      - 587
-      - 993
-      - 8000:8000
+      - "465"
```
Comment (ieugen) [inline docker-compose.yaml:22]: Use ports > 1000 so we can start it locally without root.
Users can change the ports when they deploy it.

File: conf/mailetcontainer.xml (hunk 1 above)
Comment (dleangen) [reply conf/mailetcontainer.xml:138]: Yes, you are right. I will cancel this PR and try a different approach.

**Comment by dleangen** [comment]
I will try a different approach.

**Pull request closed without merging**